### ISSUE TITLE
Issue in getting votes - Top.gg

### ISSUE THREAD

**Issue by Director0617** [dbots-pkg/dbots.js#174]
```js
const dbots = require('dbots');
const DBots = new dbots.serviceClasses.TopGG(config.TopGG_token);
const gStats = await DBots.userVoted(`678558479841230859`, message.author.id);
```

```
(node:23) UnhandledPromiseRejectionWarning: TypeError: Util_1.assert is not a function
    at TopGG.Service._request (/home/container/node_modules/dbots/lib/Interface/Service.js:100:16)
    at TopGG.userVoted (/home/container/node_modules/dbots/lib/Interface/Lists/TopGG.js:131:21)
    at Object.execute (/home/container/commands/Utility/vote.js:25:32)
    at module.exports (/home/container/events/message.js:111:9)
    at runMicrotasks (<anonymous>)
    at processTicksAndRejections (internal/process/task_queues.js:93:5)
```

**Comment by EndBug** [comment]
Hi, I've just pushed a fix that should resolve your issue: I've been able to reproduce it and the fix seems to solve it.
Please, try to install this patch and test if the error still shows up. If the problem is fixed I'll publish this in a patch version update.

```
npm install dbots-pkg/dbots.js#master
```

**Comment by Director0617** [comment]
The error still shows up.

**Comment by EndBug** [comment]
Uhm, you sure? The exact same error and stack trace? Sure you installed dbots from this branch?

**Comment by Director0617** [comment]
New error - (node:24) UnhandledPromiseRejectionWarning: Error: Request failed with status code 401

**Comment by Director0617** [comment]
Full error;
(node:24) UnhandledPromiseRejectionWarning: Error: Request failed with status code 401
    at createError (/home/container/node_modules/axios/lib/core/createError.js:16:15)
    at settle (/home/container/node_modules/axios/lib/core/settle.js:17:12)
    at IncomingMessage.handleStreamEnd (/home/container/node_modules/axios/lib/adapters/http.js:260:11)
    at IncomingMessage.emit (events.js:327:22)
    at endReadableNT (_stream_readable.js:1327:12)
    at processTicksAndRejections (internal/process/task_queues.js:80:21)

**Comment by EndBug** [comment]
That's a http 401 error: it means you're not authorized to access that endpoint

Anyway, that means that the problem reported in the issue has been solved!
Let me know if there's anything else we can do!

**Comment by Director0617** [comment]
Any way to contact you on Discord

**Comment by Director0617** [comment]
Nope nothing else

**Comment by EndBug** [comment]
Ok, regarding the issue with Top.gg: their API has been updated, so we need to update our package too.
I've added it to our to-do list, I'll keep you posted for when it'll be done 👍🏻

**Comment by Director0617** [comment]
Ok thank you

**Comment by EndBug** [comment]
Fixed in `v8.0.2` 🎉
